lib/decompress/zstd_decompress.c

```diff
@@ -1507,6 +1508,24 @@ MEM_STATIC size_t ZSTD_limitCopy(void* dst, size_t dstCapacity, const void* src,
     return length;
 }
 
+static int ZSTD_isOversized(ZSTD_DStream* zds)
```

Comment:
Although these are static functions (so they won't be published outside this translation unit), I'd prefer if these function names specified that they operate on `ZSTD_DCtx`es. Because just by looking at the name they could equally well apply to `ZSTD_CCtx`es or something else.